content/posts/multi-idiomas-com-php-sem-banco-de-dados.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Multi-idiomas com PHP sem banco de dados
+authors: Luccas Marra do Amaral
+type: post
+image: https://i.imgur.com/vApB8Ng.png
+date: 2019-01-03
+excerpt: Implemente multi-idiomas em sua aplicação PHP.
+categories:
+  - Back-end
+  - PHP
+tags:
+  - Desenvolvimento Web
+  - Multi-idiomas com PHP
+---
+
+
+
+Não é de hoje que a internet quebrou as barreiras da comunicação e é um dos maiores fatores decisivos quando queremos comprar, vender, ter entretenimento, empreender, aumentar nosso conhecimento, conhecer pessoas/lugares e realizar negócios. Levando em consideração que no mundo temos mais de 190 países, imagine quantos internautas com idiomas diferentes podemos nos comunicar.
+
+Neste artigos vamos utilizar um cenário muito recorrente. Diversos sistemas web e websites são desenvolvidos pensando somente em um único idioma e isso impossibilita que usuários estrangeiros possam navegar de maneira confortável, por isso acabam desistindo da navegação e muitas vezes diminui o interesse pelo assunto. Mas isso tem uma solução e abaixo mostrarei como ter diversos idiomas em sua aplicação PHP de forma organizada e sem a necessidade de banco de dados.
+
+
+Abaixo você encontrará 5 etapas didáticas para você implementar o multi-idiomas a sua aplicação PHP.
+ 
+
+### 1 - Organização dos arquivos
+
+Crie uma pasta “traducoes” na raiz do seu projeto e os seguintes arquivos dentro dela: 
+
+* tradutor.php
+* pt-br.php
+* en-us.php
+
+```Caso queira mais idiomas você poderá criar seguindo a mesma organização.```
```

Comment:
Coloca essa frase em itálico: "```Caso queira mais idiomas você poderá criar seguindo a mesma organização.```"

Esse arquivo vai ajudar: https://gist.github.com/JoaoTiago/0724d2eb788f6382b39476e58d821300

Mas, basicamente coloca sua frase entre _

Reply:
Corrigido